WordPress/Classes/Extensions/UITextField+TextHelper.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+extension UITextField
+{
+    func nonNilTrimmedText() -> String {
+        guard let str = text?.trim() else {
+            return ""
+        }
+        return str
+    }
```

Comment:
Interesting, I recently added `String.nonEmptyString` which is kind of the opposite approach: turn any empty string into a `nil` one. I find it a bit better to use optionals instead of empty strings as the compiler forces you to deal with the `nil` case in every occasion, but it's easy to forget handling an empty string.